lib/ledger_sync/adaptors/netsuite/vendor/ledger_serializer.rb

```diff
@@ -0,0 +1,35 @@
+# frozen_string_literal: true
+
+module LedgerSync
+  module Adaptors
+    module NetSuite
+      module Vendor
+        class LedgerSerializer < NetSuite::LedgerSerializer
+          api_resource_type :vendor
+
+          attribute ledger_attribute: :companyName,
+                    resource_attribute: :company_name
+
+          attribute ledger_attribute: :entityId,
+                    resource_attribute: :external_id
+
+          attribute ledger_attribute: :externalId,
+                    resource_attribute: :external_id
+
+          attribute ledger_attribute: :email,
+                    resource_attribute: :email
+
+          attribute ledger_attribute: :firstName,
+                    resource_attribute: :first_name
+
+          attribute ledger_attribute: :lastName,
+                    resource_attribute: :last_name
+
+          attribute ledger_attribute: :subsidiary,
+                    resource_attribute: :subsidiary,
+                    type: LedgerSerializerType::ReferenceType
```

Comment:
Just double checking, this shouldn't be the `SubsidiaryType` from above?

Reply:
Yes, that is more correct.